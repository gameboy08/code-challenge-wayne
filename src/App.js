import React from "react";
import { BrowserRouter as Router, Route } from "react-router-dom";
import { MasterPage } from "./pages";
import configureStore from "./configureStore";
import { Provider } from "react-redux";
import ErrorBoundary from "./ui/ErrorBoundary";

import "./App.scss";
function App() {
  return (
    <ErrorBoundary>
      <Provider store={configureStore()}>
        <Router>
          <Route path="/" component={MasterPage} />
        </Router>
      </Provider>
    </ErrorBoundary>
  );
}

export default App;
