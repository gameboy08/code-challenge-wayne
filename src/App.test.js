import React from "react";
import { shallow } from "enzyme";
import App from "./App";
import { shallowToJson } from "enzyme-to-json";

it("render App", () => {
  const wrapped = shallow(<App />);
  let tree = shallowToJson(wrapped);
  expect(tree).toMatchSnapshot();
});
