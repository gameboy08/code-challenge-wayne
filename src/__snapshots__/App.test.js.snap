// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render App 1`] = `
<ErrorBoundary>
  <Provider
    store={
      Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      }
    }
  >
    <BrowserRouter>
      <Route
        component={
          Object {
            "$$typeof": Symbol(react.memo),
            "WrappedComponent": [Function],
            "compare": null,
            "displayName": "Connect(MasterPage)",
            "type": [Function],
          }
        }
        path="/"
      />
    </BrowserRouter>
  </Provider>
</ErrorBoundary>
`;
